tls/s2n_signature_scheme.h

```diff
@@ -24,7 +24,7 @@
 
 struct s2n_signature_scheme {
     uint16_t iana_value;
-    const char *iana_name;
+    const char *name;
```

Comment:
These names aren't ALWAYS IANA names even rn, so I wanted to change the field name.